### PR TITLE
order_number_as_payment_reference: Changed to use the order number as…

### PR DESCRIPTION
**Pull request by tomihenttinen**
… a reference for the payment service providers instead of a random number.

### CODE REVIEW

File: src/Gateway.php [hunk 1]
```diff
@@ -1045,9 +1045,14 @@ private function set_base_payment_data($payment, $order)
         $payment->setStamp( get_current_blog_id() . '-' . $order->get_id() . '-' . time() );
 
         // Create a random reference for the order
-        $reference = sha1( uniqid( true ) );
-
         // Create a unique hash to be used as the request reference
+        //$reference = sha1( uniqid( true ) );
```
Comment (eerohakio) [inline src/Gateway.php]: Not sure do we need to keep these old things and trello references in history. What you think @loueranta ?

**Review by eerohakio — DISMISSED** [review]
One comment that requires feedback, otherwise looks okay to me 👍

File: src/Gateway.php (hunk 1 above)
Comment (loueranta) [reply src/Gateway.php]: No need to keep these, please remove them.

Comment (tomihenttinen) [reply src/Gateway.php]: Comments removed, pull request updatred.

**Review by loueranta — APPROVED** [review]

**Pull request merged**